db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
Yes, that's right. This renaming will probably crash the migration process. I'll talk with @bjrne from #62